lib/rubocop/cop/layout/line_length.rb

```diff
@@ -213,7 +213,7 @@ def excess_range(uri_range, line, line_index)
         end
 
         def max
-          cop_config['Max']
+          @max ||= cop_config['Max']
```

Comment:
I highly doubt this makes much of a difference, as `cop_config` is already cached (as it should) and the rest is just a hash lookup with a frozen string literal which is highly optimized.